src/CMakeLists.txt

```diff
@@ -20,10 +20,9 @@ if(SECP256K1_ASM STREQUAL "arm32")
   target_link_libraries(secp256k1_asm INTERFACE secp256k1_asm_arm)
 endif()
 
-# Define our export symbol only for Win32 and only for shared libs.
```

Comment:
Why remove the "shared libs" comment? That's a huge hint imo for behavior here that's quite subtle.